packages/core/src/k.ts

```diff
@@ -1,43 +1,14 @@
 import { StyledComponentProps } from "./styled";
 import { StyledProps, StyledKeyType, PseudoProps } from "@kuma-ui/system";
 
-const htmlTags: Array<Partial<keyof JSX.IntrinsicElements>> = [
-  "div",
-  "a",
-  "p",
-  "h1",
-  "h2",
-  "h3",
-  "h4",
-  "h5",
-  "header",
-  "main",
-  "footer",
-  "section",
-  "input",
-  "button",
-  "ul",
-  "ol",
-  "li",
-  "img",
-  "span",
-];
-
-type StyledComponent<T extends (typeof htmlTags)[number]> = React.FC<
+type StyledComponent<T extends keyof HTMLElementTagNameMap> = React.FC<
```

Comment:
Or, Maybe keyof JSX.IntrinsicElements is prefer instead of HTMLElementTagNameMap...?

Reply:
Or there is also keyof ReactHTML.
JSX.IntrinsicElements includes not HTML tag name.

Reply:
In my opinion, both `HTMLElementTagNameMap` and `ReactHTML` should be fine as long as they cover all HTML tags. However, in this particular case with Kuma UI, since the styled function's type is built upon keyof `JSX.IntrinsicElements`, it would be more appropriate to use `JSX.IntrinsicElements.` This ensures better consistency within the library and aligns with the existing type definitions 👍